spikingjelly/activation_based/neuron/integrate_and_fire.py

```
@@ -178,74 +178,57 @@ def neuronal_charge(self, x: torch.Tensor):
         self.v = self.v + x
 
     @staticmethod
-    def jit_eval_single_step_forward_hard_reset(
-        x: torch.Tensor, v: torch.Tensor, v_threshold: float, v_reset: float
+    def _eval_single_step_forward(
+        x: torch.Tensor, v: torch.Tensor, v_threshold: float, v_reset,
+        tau: float = None, decay_input: bool = None,
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)